src/status_im/contexts/wallet/signals.cljs

```diff
@@ -93,6 +93,9 @@
                 :dispatch [:wallet/blockchain-status-changed
                            (transforms/js->clj event-js)]}]]]}
 
+       "wallet-blockchain-health-changed"
+       {:fx [[:dispatch [:wallet/blockchain-health-changed (transforms/js->clj event-js)]]]}
+
```

Comment:
With this, I believe we can safely remove the `wallet/blockchain-status-changed` event. 

cc @shivekkhurana

Reply:
I don't want to do that here because it means additional load in this PR. But happy to create another issue + work on it later on.